docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,1163 @@
+{
```

Comment:
<p>I think the overview table is very useful, but maybe move it to <code>docs/datasets.rst</code>?  And put a link here instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>